### PR TITLE
doc: wifi: Add Optimizing scan operation page to Developing ug

### PR DESCRIPTION
**Pull request by richabp**
Add a new page on Optimizing scan operation to the Developing with nRF70 Series user guide.

SHEL-1757

### CODE REVIEW

**Review by krish2718 — COMMENTED** [review]
Should the signoff also include original authors?

**Comment by peknis** [comment]
Should some of this belong to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize/optimizing/memory.html?

**Comment by richabp** [comment]
> Should the signoff also include original authors?

I think its a documentation page hence not required.
@sachinthegreen: Can you confirm?

**Comment by sachinthegreen** [comment]
> > Should the signoff also include original authors?
> 
> I think its a documentation page hence not required. @sachinthegreen: Can you confirm?

I am not aware of the protocol here but seems ok to me that the signoff of original authors also should be included. In this case it would be @udaynordic , @sachinthegreen and @krish2718

File: doc/nrf/device_guides/working_with_nrf/nrf70/developing/scan_operation.rst [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - Configuration parameters
+   * - Band filtering
+     - | Build time:
+       | :kconfig:option:`CONFIG_WIFI_MGMT_SCAN_BANDS`
+       | Run time:
+       | :c:enumerator:`bands`
```
Comment (sachinthegreen) [inline doc/nrf/device_guides/working_with_nrf/nrf70/developing/scan_operation.rst:131]: Shouldn't these be clickable links (in this case pointing to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/connectivity/networking/api/wifi.html#c.wifi_scan_params.bands)

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

**Comment by richabp** [comment]
> Should some of this belong to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize/optimizing/memory.html or https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize/optimizing/power.html?
> 
> We could maybe use some of the power optimization stuff as includes in the Power optimization section.

@sachinthegreen: Can you please advise?

**Comment by sachinthegreen** [comment]
> > Should some of this belong to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize/optimizing/memory.html or https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize/optimizing/power.html?
> > We could maybe use some of the power optimization stuff as includes in the Power optimization section.
> 
> @sachinthegreen: Can you please advise?

I don't have a opinion on this. Maybe @ioannisg can advise.

File: doc/nrf/device_guides/working_with_nrf/nrf70/developing/scan_operation.rst [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+* Faster scan times
+
+* Reduced power consumption
+
+Scan API
+********
+
+The scan API can be used to implement features such as scan functionality and controlled scan provided by the nRF70 Series ICs.
+
+You can find the scan API reference in :ref:`zephyr:wifi_mgmt` in the Zephyr documentation.
```
Comment (sachinthegreen) [inline doc/nrf/device_guides/working_with_nrf/nrf70/developing/scan_operation.rst:81]: This should be pointing to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12233/zephyr/connectivity/networking/api/wifi.html#c.wifi_mgmt_ops.scan

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

**Comment by krish2718** [comment]
> > > Should the signoff also include original authors?
> > 
> > 
> > I think its a documentation page hence not required. @sachinthegreen: Can you confirm?
> 
> I am not aware of the protocol here but seems ok to me that the signoff of original authors also should be included. In this case it would be @udaynordic , @sachinthegreen and @krish2718

FYI, this would be adding below lines to the commit.

```
Signed-off-by: Sachin D Kulkarni <sachin.kulkarni@nordicsemi.no>
Signed-off-by: Uday Kannemadugu <Uday.Kannemadugu@nordicsemi.no>
Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>
```
This acts as both legal ownership and also to track original authors.

**Review by kittydepa — CHANGES_REQUESTED** [review]
Small suggested changes

**Review by kittydepa — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Comment by ioannisg** [comment]
I have a feeling that the diagrams will be easier to read if the X-axis text is written in the opposite direction, that is as the Y-axis text.

**Comment by richabp** [comment]
> I have a feeling that the diagrams will be easier to read if the X-axis text is written in the opposite direction, that is as the Y-axis text. At least, for me it seems harder to read the bar labels as they are written now.

This is as per our style guide: https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120293046/Figure+guide. I have changed the direction of the text on the Y-axis.

@mirjaku : Can you provide guidance here?

**Comment by ioannisg** [comment]
> @richabp @ioannisg The guideline to write vertical diagram texts from top to bottom has been in effect and implemented consistently in our diagrams for the better part of a decade, so for the sake of stylistic consistency I would not make an exception for this one diagram.

There needs to be an exception to this rule, IMHO, at least for this particular type of bar-graphs with lots of long texts in the x-axis, otherwise such graphs are very hard to read.

**Pull request merged**